_posts/2023-01-13-adbc-0.1.0-release.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Apache Arrow ADBC 0.1.0 (Libraries) Release"
+date: "2023-01-13 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 0.1.0 release of the
+Apache Arrow ADBC libraries. This covers includes [**63 resolved
+issues**][1] from [**8 distinct contributors**][2].
+
+This is a release of the **libraries**, which are at version 0.1.0.
+The **API specification** is versioned separately and is at version
+1.0.0.
```

Comment:
This might be a good place to add a link to the Introducing ADBC blog post for readers interested in learning more about the specification